ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/Config.scala

```diff
@@ -362,6 +364,14 @@ object Config {
             "Enable the development version of the Daml-LF language. Highly unstable. Should not be used in production."
           )
 
+        opt[Unit]("index-append-only-schema-unsafe")
+          .optional()
+          .hidden()
+          .text(
+            s"Enable the use of the experimental append-only index database schema. Highly unstable. Should not be used in production."
```

Comment:
Nit: this may be shortened as
```suggestion
            s"Use the experimental append-only index database schema. Highly unstable. Should not be used in production."
```
or
```suggestion
            s"Enable the experimental append-only index database schema. Highly unstable. Should not be used in production."
```